simapp/v2/simdv2/cmd/root_di.go

```diff
@@ -43,7 +42,7 @@ func NewRootCmd() *cobra.Command {
 			simapp.AppConfig(),
 			depinject.Supply(
 				log.NewNopLogger(),
-				simtestutil.NewAppOptionsWithFlagHome(tempDir()),
+				simtestutil.NewAppOptionsWithFlagHome(""), // TODO: currently work with empty home
```

Comment:
try home hack since we remove `tempDir()`. Don't know how it still works or maybe wrong somewhere that I didn't see